cmd/fleet/main.go

```diff
@@ -62,4 +61,11 @@ func applyDevFlags(cfg *config.FleetConfig) {
 	cfg.Mysql.Username = "fleet"
 	cfg.Mysql.Database = "fleet"
 	cfg.Mysql.Password = "insecure"
+
+	if cfg.Prometheus.BasicAuth.Username == "" {
+		cfg.Prometheus.BasicAuth.Username = "fleet"
+	}
+	if cfg.Prometheus.BasicAuth.Password == "" {
+		cfg.Prometheus.BasicAuth.Password = "insecure"
+	}
```

Comment:
Github probably hided it, but this was added to `applyDevFlags` function.

Reply:
Ahhhh, that makes total sense!

Reply:
This is so that when running with `--dev` the `/metrics` are always enabled (and you don't need to explicitly specify username and password on env/config).

(This also means that on `--dev` mode we cannot really disable `/metrics`...)